Consul/Filtering/Selectors/ServiceSelector.cs

```diff
@@ -29,6 +29,9 @@ public sealed class ServiceSelector : Selector
         public MetaSelector Meta { get; } = new MetaSelector(Self);
 
         public override string Encode() => Self;
+
+        public static Filter operator ==(ServiceSelector selector, string value) => Filters.Eq(selector, value);
```

Comment:
I don't think a "service" is a valid selector that can be used for anything. Also, this is not being used in an actual test (apart from encoding tests). I think that it should be removed.

Reply:
Hi.
The 'Service' selector is used in the following api queries:
https://developer.hashicorp.com/consul/api-docs/agent/service#list-services
https://developer.hashicorp.com/consul/api-docs/catalog#retrieve-map-of-services-for-a-node
https://developer.hashicorp.com/consul/api-docs/catalog#list-services-for-node

Reply:
Ok, I see. It would be great though if you implement at least one test case which uses this APIs.